instantout/instantout.go

```diff
@@ -213,7 +213,7 @@ func (i *InstantOut) createSweeplessSweepTx(feerate chainfee.SatPerKWeight) (
 
 	// Estimate the fee
 	weight := sweeplessSweepWeight(len(inputReservations))
-	fee := feerate.FeeForWeight(weight)
+	fee := feerate.FeeForWeight(lntypes.WeightUnit(weight))
```

Comment:
I propose `sweeplessSweepWeight` to return `lntypes.WeightUnit`.

`sweeplessSweepWeight` converts `lntypes.WeightUnit` to int64 as its last action:
```go
return int64(weightEstimator.Weight())
```

I propose to return `weightEstimator.Weight()` and to avoid int64 step here.